scripts/postBuild.ts

```diff
@@ -0,0 +1,5 @@
+import {promises} from 'fs';
+
+promises.copyFile('README.md', 'dist/README.md')
+    .then(() => console.log('README.md was copied to dist'))
+    .catch(err => { if (err) throw err;});

```

Comment:
In this project we prefer to use `async`/`await` syntax for dealing with promises. Please consider refactoring your code to the following

```ts
async main() {
  try {
    await fs.copyFile('README.md', 'dist/README.md');
    console.log('README.md was copied to dist');
  } catch (e) {
    console.error(e);
  }
}

main();
```